src/main/java/org/elasticsearch/action/admin/cluster/health/ClusterHealthResponse.java

```diff
@@ -220,6 +234,11 @@ public void readFrom(StreamInput in) throws IOException {
                 validationFailures.add(in.readString());
             }
         }
+
```

Comment:
I am going to try and see if this can be back ported to 1.6, if not, I will remove it afterwards